config.json

```diff
@@ -580,3 +580,4 @@
         }
     ]
 }
+{"mode":"full","isActive":false}
```

Comment:
This line should be removed

Reply:
@ShobhitAd, thanks. done in https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/250/commits/3898f8939b0732d8bc66a22be3c187270144db53